daml-lf/spec/daml-lf-1.rst

```diff
@@ -3030,6 +3219,69 @@ as described by the ledger model::
    —————————————————————————————————————————————————————————————————————— EvUpdEmbedExpr
      'embed_expr' @τ e ‖ (st; keys)  ⇓ᵤ  ur
 
+     e₁  ⇓  Err v₁
+     e₂[x ↦ v₁]  ⇓  Err v₂
+   —————————————————————————————————————————————————————————————————————— EvUpdTryCatchErr1e
+     'try' @τ e₁ 'catch' x. e₂ ‖ S₀  ⇓ᵤ  (Err v₂, 'fail' v₁ ε)
+
+     e₁  ⇓  Ok u₁
+     u₁ ‖ S₀  ⇓ᵤ  (Err v₁, tr₁)
+     e₂[x ↦ v₁]  ⇓  Err v₂
+   —————————————————————————————————————————————————————————————————————— EvUpdTryCatchErr1u
+     'try' @τ e₁ 'catch' x. e₂ ‖ S₀  ⇓ᵤ  (Err v₂, 'fail' v₁ tr₁)
+
+     e₁  ⇓  Err v₁
+     e₂[x ↦ v₁]  ⇓  Ok ('None' @_)
```

Comment:
What is `@_` ? Is it a "matcher' for any type. Again I prefer you follow the existing convention to use a singleton variable or to change the file to not use any-more singleton variable in inference rule.

Reply:
Oops, that was a short-cut that made it into the final version. Changed it to `@σ`, where `σ` is an otherwise not mentioned type.